get.go

```diff
@@ -69,18 +69,23 @@ func init() {
 	}
 }
 
+// DefaultClient is the default client used
+// for all requests.
+var DefaultClient = &Client{
```

Comment:
This should be changed to a function to effectively replicate the functionality of the previous implementation.

This creates a singleton object, each other call was declaring a new `Client` of its own.